### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by hydrabeer**
Updated to meet almost every requirement for the README. Still need to add more images and ensure comprehensive, precise, and complete feature and usage descriptions.

### CODE REVIEW

**Review by saeedshrouf — APPROVED** [review]
Very solid ReadMe update, looks good - formatting in MD done well.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,115 @@
...
+passwords for each of your accounts. Doorkey makes it easy to generate and store these passwords, so 
+you can stay safe online.
+
+## Features
+### Local and Cloud Storage
+Create a local vault to store your credentials on your device, or use a cloud vault to access your
+credentials from anywhere.
+### Password Generation
+Generate secure, random passwords with Doorkey's password generator. Customize the length,
+character set, and other parameters to satisfy the requirements of any website.
```
Comment (HACKERALERT) [inline README.md:43]: You could also talk about the atomispheric pressure part

**Review by HACKERALERT — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HACKERALERT) [reply README.md:43]: I will do this in a PR to this PR branch.

**Comment by HACKERALERT** [comment]
Nice job! I'll continue to improve the readme here: https://github.com/hydrabeer/doorkey/pull/55

**Pull request merged**